tests/testthat/test-repair_names.R

```diff
@@ -0,0 +1,52 @@
+context("repair_names")
+
+test_that("repair column names when none are provided", {
+    dat <- as_data_frame(diag(3))
```

Comment:
This is not a particularly good initial value, because it's currently wrong and might change. Something like `data_frame(a = 1, b = 2, c = 3)` will do fine here.